incubator/metrics-http/src/test/java/io/aklivity/zilla/runtime/metrics/http/internal/HttpMetricsTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.metrics.http.internal;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.metrics.CollectorContext;
+import io.aklivity.zilla.runtime.engine.metrics.Metric;
+import io.aklivity.zilla.runtime.engine.metrics.Metrics;
+import io.aklivity.zilla.runtime.engine.metrics.MetricsContext;
+
+public class HttpMetricsTest
+{
+    @Test
+    public void shouldResolveHttpRequestSize()
+    {
+        Configuration config = new Configuration();
+        Metrics metrics = new HttpMetrics(config);
+        CollectorContext httpCollectorContext = new HttpCollectorContext();
```

Comment:
This one can be called `collector` and should be created as a `mock`.

See example https://github.com/aklivity/zilla/blob/0e168af10f862cabc8cd4554281af8637808cbc0/runtime/guard-jwt/src/test/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardTest.java#L64